### PR TITLE
[GSoC] Added option to set a new avatar

### LINKED ISSUE #3826: [Leaderboard] Get a user's avatar
Commons does not really have a notion of avatar, but people who want to have a representative picture post it to their userpage. Examples:

https://commons.wikimedia.org/wiki/User:%C5%A0J%C5%AF (self-portrait)
https://commons.wikimedia.org/wiki/User:Amqui (self-upload picture showing the user's nickname)
https://commons.wikimedia.org/wiki/User:Giantflightlessbirds (picture the user is most proud of)
https://commons.wikimedia.org/wiki/User:Dan_Koehl (self-protrait in an infobox within a very elaborate userpage)
https://commons.wikimedia.org/wiki/User:Darren_J._Prior (several pictures that the user is proud of, we can just use the first one)

In most cases though there is no such picture, example: https://commons.wikimedia.org/wiki/User:Jeuwre

I suggest using the API that fetches media of a page, I don't remember its name but it was discussed recently by Jo I think.

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Fixes #3826 

**Tests performed**
Tested prodDebug on Google Pixel 2 with API level 29.

**Screenshots**
Screenshot_1596473084
Screenshot_1596473086
Screenshot_1596473088
Screenshot_1596473100

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -656,4 +656,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="leaderboard_column_rank">Rank</string>
   <string name="leaderboard_column_user">User</string>
   <string name="leaderboard_column_count">Count</string>
+  <string name="setting_avatar_dialog_title">Set as Leaderboard Avatar</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:659]: Lowercase: Set as leaderboard avatar

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -656,4 +656,9 @@ Upload your first media by tapping on the add button.</string>
...
+  <string name="setting_avatar_dialog_message">Setting as Avatar, please wait</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:660]: Lowercase: avatar

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -656,4 +656,9 @@ Upload your first media by tapping on the add button.</string>
...
+  <string name="avatar_set_successfully">Avatar Set Successfully</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:661]: Lowercase and no need to say "successfully": Avatar set

**Comment by madhurgupta10** [comment]
@nicolas-raoul Changes made https://github.com/commons-app/apps-android-commons/pull/3897

**Pull request merged**